scripts/test/hyriseBenchmarkCore.py

```diff
@@ -26,16 +28,21 @@ def check_json(json, argument, error, return_error, difference=None):
             return True
     else:
         if abs(json - argument) > difference:
-            print("ERROR: " + error + " " + str(json) + " " + str(argument) + " " + str(difference))
+            print(f"ERROR: {error} {json} {argument} {difference}")
             return True
     return return_error
 
 
-def initialize(arguments, benchmark_name, verbose):
+def initialize():
     if len(sys.argv) == 1:
-        print("Usage: ./scripts/test/" + benchmark_name + "_test.py <build_dir>")
+        print("Usage: ./scripts/test/{BENCHMARK_NAME}_test.py <build_dir>")
```

Comment:
What is BENCHMARK_NAME here? If it is a placeholder that's meant to be printed, why not use the same syntax as for build_dir?

Reply:
Sorry, forgot to remove this placeholder. Fixed now.